src/components/ComicList.tsx

```diff
@@ -1,13 +1,33 @@
 import { useNavigate } from "react-router-dom";
-import VelhonTaloudenhoitajaEtusivu from "../../src/assets/VelhonTaloudenhoitaja/velhontaloudenhoitaja-etusivu.png";
-import SiivetonLepakkoEtusivu from "../../src/assets/SiivetonLepakko/varsinainen-etusivu.png";
+import { useState, useEffect } from "react";
+import { apiBaseUrl } from "../constants";
+import comicService from "../services/comicService";
+
 import "./ComicPage.css";
 
 const ComicList = () => {
   const navigate = useNavigate();
+  const [SiivetonLepakkoEtusivu, setSiivetonLepakkoEtusivu] = useState("");
```

Comment:
I whould have `comicFrontPages` variable. 

- get all front pages
- loop front pages

Separate useState for each comic is not very good to me.

Reply:
I don't know if I changed it for better or for worse, please look at it again!

Reply:
How about something like this? 
![image](https://github.com/user-attachments/assets/b1d51569-9d38-485f-9068-c7e93453f0e3)

Reply:
`comicInfo` thing is not so important, made it just to be able to loop over many comics front page infos.
More important `promise.all`, if you have to make many ruequests it good to make them all at once not one by one.